src/model/Space.cpp

```diff
@@ -2827,14 +2831,10 @@ namespace model {
     std::vector<Point3d> Space_Impl::floorPrint() const {
       double tol = 0.01;  // 1 cm tolerance
 
-      // get all surfaces, sort so results are repeatable
-      std::vector<Surface> surfaces = this->surfaces();
-      std::sort(surfaces.begin(), surfaces.end(), IdfObjectNameLess());
-
       // find all floors
       boost::optional<double> z;
       std::vector<Surface> floors;
-      for (const Surface& surface : surfaces) {
+      for (const Surface& surface : this->surfaces()) {
```

Comment:
I can see how floorPrint *could* be affected by order of surfaces, since it's doing geometry things with it, but the original issue says it's floorArea that was affected, and as I said at https://github.com/NREL/OpenStudio/issues/5465#issuecomment-3179744342 I don't see how that's possible.